.travis.yml

```diff
@@ -18,6 +18,8 @@ jobs:
       env: ARCH=ppc64
     - name: "ARCH=ppc64le LD=ld.lld"
       env: ARCH=ppc64le LD=ld.lld-11
+    - name: "ARCH=s390"
```

Comment:
can you wire this up so that we explicitly put `BOOT=0` or something, such that it's explicit in the config that we don't boot test?  That will make it more obvious in the build results, and we won't have to make further changes to driver.sh when turning on builds of hexagon.

Reply:
Sure, done.